packages/core/src/utils/stripTypename.ts

```diff
@@ -0,0 +1,21 @@
+export const stripTypename = (data: any): any => {
+  if (!data || typeof data !== 'object') return data;
+
+  return Object.keys(data).reduce((acc, key: string) => {
```

Comment:
Wondering if there's a scenario where data is an array from the start

Reply:
No, since we'd only apply this to things that are objects in GraphQL (results / variables / etc) we'd never start with an array